### PR TITLE
x86_cpu_flags:new case to test mce=off

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1903534

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (95.25 s)

**Comment by nanliu-r** [comment]
@huangyum @vivianQizhu Please help to review this, thanks.

File: qemu/tests/cfg/x86_cpu_flags.cfg [hunk 1]
```diff
@@ -84,3 +84,6 @@
                     check_clock = 'cat /sys/devices/system/clocksource/clocksource0/available_clocksource | grep tsc'
                     cpu_model_flags += ",+invtsc"
                     flags = "nonstop_tsc"
+                - mce:
+                    cpu_model_flags += ",mce=off"
+                    flags = ""
```
Comment (huangyum) [inline qemu/tests/cfg/x86_cpu_flags.cfg:89]: I guess you need add `no_flags = "mce"`.

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg:89]: > I guess you need add `no_flags = "mce"`.

Thanks and update.
 (1/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (97.70 s)
 (2/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (71.75 s)
 (3/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (96.10 s)
 (4/4) type_specific.io-github-autotest-qemu.x86_cpu_flags.others.mce: PASS (155.41 s)

**Review by huangyum — APPROVED** [review]
ACK

**Comment by nanliu-r** [comment]
@vivianQizhu requests your approval, thanks.

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**